client/src/components/Billing/index.js

```diff
@@ -0,0 +1,183 @@
+      "Help documentation",
+      "Priority email support"
+    ],
+    buttonText: "Join us",
+    buttonVariant: "outlined"
+  }
+];
+
+function Pricing(props) {
+  const { classes } = props;
+
+  return (
+    <React.Fragment>
+      <CssBaseline />
+
+      <main className={classes.layout}>
+        {/* Hero unit */}
+        <div className={classes.heroContent}>
+          <Typography
+            component="h1"
+            variant="h2"
+            align="center"
+            color="textPrimary"
+            gutterBottom
+          >
+            Pricing
+          </Typography>
+          <Typography
+            variant="h6"
+            align="center"
+            color="textSecondary"
+            component="p"
+          >
+            Pickem is a investor application that makes it easier to research
+            stocks and make better investment decisions. Become a better
+            investor today by picking one of our tiered plans below.
+          </Typography>
+        </div>
+        {/* End hero unit */}
+        <Grid container spacing={40} alignItems="flex-end">
+          {tiers.map(tier => (
+            // Pro card is full width at sm breakpoint
+            <Grid
+              item
+              key={tier.title}
+              xs={12}
+              sm={tier.title === "Pro" ? 12 : 6}
+              md={4}
+            >
+              <Card>
+                <CardHeader
+                  title={tier.title}
+                  subheader={tier.subheader}
+                  titleTypographyProps={{ align: "center" }}
+                  subheaderTypographyProps={{ align: "center" }}
+                  action={tier.title === "Investor" ? <StarIcon /> : null}
+                  className={classes.cardHeader}
+                />
+                <CardContent>
+                  <div className={classes.cardPricing}>
+                    <Typography component="h2" variant="h3" color="textPrimary">
+                      ${tier.price}
+                    </Typography>
+                    <Typography variant="h6" color="textSecondary">
+                      /mo
+                    </Typography>
+                  </div>
+                  {tier.description.map(line => (
+                    <Typography variant="subtitle1" align="center" key={line}>
+                      {line}
+                    </Typography>
+                  ))}
+                </CardContent>
+                <CardActions className={classes.cardActions}>
+                  <Button
+                    fullWidth
+                    variant={tier.buttonVariant}
+                    color="primary"
+                  >
+                    {tier.buttonText}
+                  </Button>
+                </CardActions>
+              </Card>
+            </Grid>
+          ))}
+        </Grid>
+      </main>
+    </React.Fragment>
+  );
+}
+
+Pricing.propTypes = {
+  classes: PropTypes.object.isRequired
+};
+
+export default withStyles(styles)(Pricing);
```

Comment:
The name of this file should be something other than `index.js` since it's not actually the billing view. I think it's fine to leave in the Billing folder but I would name it `Pricing.js` instead.